pyuvdata/uvcal/uvcal.py

```diff
@@ -3265,16 +3191,19 @@ def __add__(
                 taxis_num: {"inds": tnew_inds, "order": t_order},
                 jaxis_num: {"inds": jnew_inds, "order": j_order},
             }
+            for axis, subdict in axis_dict.items():
+                for name, param in zip(this._data_params, this.data_like_parameters):
```

Comment:
Having the loops in this order is critical because otherwise the `param` gets reused without updating on each subsequent axis. I lost a day and a half to this bug.